### PR TITLE
Tightened Spotify profile

### PR DESCRIPTION
**Pull request by Fred-Barclay**
This should tighten the Spotify profile a good bit. Mainly I added a `private-etc` filter and blacklisted unneeded files and directories. (Some of these directories are already read-only but why even expose them since they're not necessary?)

### CODE REVIEW

File: etc/spotify.profile [hunk 1]
```diff
@@ -27,5 +24,24 @@ protocol unix,inet,inet6,netlink
...
+blacklist /lost+found
+blacklist /media
+blacklist /mnt
+blacklist /opt
+blacklist /root
+blacklist /sbin
+blacklist /srv
+blacklist /sys
+blacklist /var
+blacklist /initrd.img
```
Comment (reinerh) [inline etc/spotify.profile]: I would put those 4 files to disable-common.inc.

Comment (Fred-Barclay) [inline etc/spotify.profile]: I've thought about that but there may be some program that depend 'em that also use disable-common.inc. I need to do some testing first. :+1:

Comment (reinerh) [inline etc/spotify.profile]: They are usually only written on kernel upgrades (or when the initramfs is rebuilt), and read by bootloaders.

Comment (Fred-Barclay) [inline etc/spotify.profile]: Okay, that's a great idea then. I'll do some testing (I'm trying to rework the disable-* files anyways) and then I'll send in a PR, unless you'd rather do it. :question:

Comment (Fred-Barclay) [inline etc/spotify.profile]: What about /vmlinuz? Wouldn't it be a good idea to blacklist it too in disable-common.inc?

**Comment by netblue30** [comment]
Merged, thanks.

**Pull request merged**